docs/install.md

```diff
@@ -54,221 +25,122 @@ following:
+    will delete all uncommited changes.
 
-1.  Ensure that the Rust toolchain can compile code for ARM devices.
-
-### Replacing the certificates
-
-All the generated certificates and private keys are stored in the directory
-`crypto_data/`.
-
-This is the expected content after running our `setup.sh` script:
-
-File                     | Purpose
------------------------- | --------------------------------------------------------
-`aaguid.txt`             | Text file containaing the AAGUID value
-`opensk_ca.csr`          | Certificate sign request for the Root CA
-`opensk_ca.key`          | ECC secp256r1 private key used for the Root CA
-`opensk_ca.pem`          | PEM encoded certificate of the Root CA
-`opensk_ca.srl`          | File generated by OpenSSL
-`opensk_cert.csr`        | Certificate sign request for the attestation certificate
-`opensk_cert.pem`        | PEM encoded certificate used for the authenticator
-`opensk.key`             | ECC secp256r1 private key used for the autenticator
-`opensk_upgrade.key`     | Private key for signing upgrades through CTAP
-`opensk_upgrade_pub.pem` | Public key added to the firmware for verifying upgrades
-
-If you want to use your own attestation certificate and private key, simply
-replace `opensk_cert.pem` and `opensk.key` files.
-
-Our build script `build.rs` is responsible for converting the `aaguid.txt` file
-into raw data that is then used by the Rust file `src/ctap/key_material.rs`.
+    ```shell
+    ./reset.sh
+    ./setup.sh
+    ```
 
-Our configuration script `tools/configure.py` is responsible for configuring
-an OpenSK device with the correct certificate and private key.
+*   Flash your board according to the
+    [flashing instructions below](#Flashing-a-firmware]. If you come from an
+    OpenSK version before the 2.0 certified one, your credential storage is not
+    backwards compatible and you have to reset it. :warning: You will lose
+    logins to all websites that you registered with OpenSK. To erase your
+    persistent storage, run the deploy script twice: Once with the application
+    parameter `--erase_storage`, and once with `--opensk` as usual.
 
-### Flashing a firmware
+    This reset also clears the certificate. For a privacy discussion, see the
+    [certificate section in Customization](customization.md#Certificate-considerations).
+    If you want to reinstall it, you also need to rerun:
 
-#### Nordic nRF52840-DK board
+    ```shell
+    ./tools/configure.py \
+        --certificate=crypto_data/opensk_cert.pem \
+        --private-key=crypto_data/opensk.key
+    ```
 
-![Nordic development kit](img/devkit_annotated.jpg)
+#### Initial setup
 
-1.  Connect a micro USB cable to the JTAG USB port.
+To clone and setup the repository for the develop branch, run the following
+commands:
 
-1.  Run our script for compiling/flashing Tock OS and OpenSK on your device
-    (_output may differ_):
+```shell
+git clone https://github.com/kaczmarczyck/OpenSK.git
```

Comment:
Wrong repository ;)

Reply:
Oops, and also merged in the `git checkout develop` for brevity.